boards/shields/weact_ov2640_cam_module/weact_ov2640_cam_module.overlay

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) Copyright (c) 2024 Charles Dias
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/ {
+	chosen {
+		zephyr,camera = &zephyr_camera_dvp;
+	};
+};
+
+&zephyr_camera_i2c {
+	status = "okay";
+	clock-frequency = <I2C_BITRATE_FAST>;
+
+	ov2640: ov2640@30 {
+		compatible = "ovti,ov2640";
+		reg = <0x30>;
+		status = "okay";
+
+		port {
+			ov2640_ep_out: endpoint {
+				remote-endpoint = <&dcmi_ep_in>;
+			};
+		};
+	};
+};
+
+&zephyr_camera_dvp {
+	status = "okay";
+	sensor = <&ov2640>;
+	bus-width = <8>;
+	hsync-active = <0>;
+	vsync-active = <0>;
+	pixelclk-active = <1>;
+	capture-rate = <1>;
+
+	port {
+		dcmi_ep_in: endpoint {
```

Comment:
Maybe this would need an alias to be as generic as the `zephyr_camera_dvp`? `zephyr_camera_dvp_ep_in`? Or less verbose `zephyr_camera_dvp_in`? Or something else?